source/chapters/user_interface.rst

```diff
@@ -905,6 +911,9 @@ on the left end of the crossfader and vice versa. This is also known as
 *Hamster Style*. To adjust the crossfader to your style of mixing, go to
 :menuselection:`Preferences --> Crossfader`.
 
+The crossfader orientation sets the channel's crossfader orientation.
+It can be set to the left-side, right-side or center of the crossfader and it is unaffected by the :term:`crossfader`.
```

Comment:
```suggestion
The :term:`crossfader` orientation determines to which side of the crossfader the deck is assigned.
It can be set to the left or right side, or to the center where it is unaffected by the crossfader knob.
```